packages/datepickers/src/styled/styled-datepicker.tsx

```diff
@@ -52,36 +54,40 @@ export const StyledHeaderPaddle = styled.div<{
   display: flex;
   justify-content: center;
   align-items: center;
-  color: ${PALETTE.grey[600]};
+  color: ${props => getColor('neutralHue', 600, props.theme)};
   cursor: pointer;
   border-radius: 50%;
 
   ${props => props.isHidden && `visibility: hidden;`};
 
   :hover {
-    color: ${PALETTE.grey[800]};
-    background-color: ${rgba(PALETTE.blue[600], 0.08)};
+    color: ${props => getColor('foreground', 800, props.theme)};
+    background-color: ${props => getColor('primaryHue', 600, props.theme, 0.08)};
   }
 
   :active {
-    background-color: ${rgba(PALETTE.blue[600], 0.2)};
-    color: ${PALETTE.grey[800]};
+    background-color: ${props => getColor('primaryHue', 600, props.theme, 0.2)};
+    color: ${props => getColor('foreground', 800, props.theme)};
```

Comment:
Never use `getColor` for `foreground` – only `props.theme.colors.foreground`. But in this case, I think it's a variation on `neutralHue`. But we can go over together.

Reply:
Was hoping to have this discussion above with my opening question. We will need to have a 100-900 scale with the `foreground` color to accurately theme icon buttons and other content which uses the "text" colors.

Will move to neutral for now.